tests/selenium/src/tests/50-infoblock-desc.mjs

```diff
@@ -0,0 +1,26 @@
+import actionOpenMap from '../actions/actionOpenMap.mjs';
+import { clickBy, enclose, matchTextBy, waitBy } from '../lib.mjs';
+import { By } from 'selenium-webdriver';
+import { resolve } from 'node:path';
+import actionIdleWait from '../actions/actionIdleWait.mjs';
+import { driver } from '../options.mjs';
+
+const TEST_GPX_FILE = 'test-infoblock-desc.gpx';
+const DESC = '1 test\n' + '2 test\n' + '3 test\n' + 'www.osmand.net';
+export default async function test() {
+    await actionOpenMap();
+
+    await clickBy(By.id('se-show-main-menu'), { optional: true });
+    await clickBy(By.id('se-show-menu-tracks'), { optional: true });
```

Comment:
you can remove this 2 lines if you switch upload to action: actionUploadGpx() (see test example 20-local-to-cloud)